hack/ci/ci-bootstrap.yaml

```diff
@@ -65,10 +64,59 @@ files:
           CI_URL="gs://kubernetes-release-dev/ci/$CI_VERSION-bazel/bin/linux/amd64"
 
           for CI_PACKAGE in "${PACKAGES_TO_TEST[@]}"; do
-            echo "* downloading package: $CI_URL/$CI_PACKAGE.$PACKAGE_EXT"
-            ${GSUTIL} cp "$CI_URL/$CI_PACKAGE.$PACKAGE_EXT" "$CI_DIR/$CI_PACKAGE.$PACKAGE_EXT"
-            ${SUDO} dpkg -i "$CI_DIR/$CI_PACKAGE.$PACKAGE_EXT" || echo "* ignoring expected 'dpkg -i' result"
+            echo "* downloading package: $CI_URL/$CI_PACKAGE"
+            ${GSUTIL} cp "$CI_URL/$CI_PACKAGE" "$CI_DIR/$CI_PACKAGE"
+            chmod +x "$CI_DIR/$CI_PACKAGE"
+            mv "$CI_DIR/$CI_PACKAGE" "/usr/bin/$CI_PACKAGE"
           done
+
+          echo "configuring kubelet service"
+
+          mkdir -p /lib/systemd/system/
+          cat <<EOF >> /lib/systemd/system/kubelet.service
+[Unit]	
+Description=kubelet: The Kubernetes Node Agent	
+Documentation=http://kubernetes.io/docs/	
+
+[Service]	
+ExecStart=/usr/bin/kubelet	
+Restart=always	
+StartLimitInterval=0	
+RestartSec=10	
+
+[Install]	
+WantedBy=multi-user.target
+EOF
+          
+          mkdir -p /etc/systemd/system/kubelet.service.d/
+          cat <<EOF >> /etc/systemd/system/kubelet.service.d/10-kubeadm.conf
+# Note: This dropin only works with kubeadm and kubelet v1.11+	
+[Service]	
+Environment="KUBELET_KUBECONFIG_ARGS=--bootstrap-kubeconfig=/etc/kubernetes/bootstrap-kubelet.conf --kubeconfig=/etc/kubernetes/kubelet.conf"	
+Environment="KUBELET_CONFIG_ARGS=--config=/var/lib/kubelet/config.yaml"	
+# This is a file that "kubeadm init" and "kubeadm join" generate at runtime, populating the KUBELET_KUBEADM_ARGS variable dynamically	
+EnvironmentFile=-/var/lib/kubelet/kubeadm-flags.env	
+# This is a file that the user can use for overrides of the kubelet args as a last resort. Preferably, the user should use	
+# the .NodeRegistration.KubeletExtraArgs object in the configuration files instead. KUBELET_EXTRA_ARGS should be sourced from this file.	
+EnvironmentFile=-/etc/default/kubelet	
+ExecStart=	
+ExecStart=/usr/bin/kubelet $KUBELET_KUBECONFIG_ARGS $KUBELET_CONFIG_ARGS $KUBELET_KUBEADM_ARGS $KUBELET_EXTRA_ARGS
+EOF
+
+          mkdir -p /usr/lib/modules-load.d/
+          cat <<EOF >> /usr/lib/modules-load.d/kubeadm.conf 
+# Load br_netfilter module at boot	
+br_netfilter
+EOF
+
+          mkdir -p /etc/sysctl.d/ 
+          cat <<EOF >> /etc/sysctl.d/50-kubeadm.conf 
+# The file is provided as part of the kubeadm package	
+net.ipv4.ip_forward = 1
+EOF
+
+          systemctl enable kubelet
+          systemctl start kubelet
```

Comment:
The systemd unit files, modules, sysctl, and service configuration should already be configured in the base image that is used.

Reply:
Just to double-check, does the base image come with an existing kubeadm package already installed? then, is it right to assume the goal of this change is only to override the binaries?

Reply:
Correct, the default image id that is used is pulled from the published CAPA images: https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/release-0.4/hack/ci/e2e-conformance.sh#L220-L230, so only overriding the binaries *should* be needed.

Reply:
fixed. then it is simpler than expected 😄